### DISCUSSION TITLE
Garage dropdown for button "Send to garage"

### DISCUSSION THREAD

**Discussion by Frozen-Rainbow** [voxeltycoon/issues#546] [UI&UX]
It would be much more convenient if a drop-down of the garages appears by clicking on the "Send to garage" button instead of searching a garage on the map.
The garaged should be sorted by the distance to the vehicles current position.

**Comment by JezzyBelle** [comment]
LOL...  Yep. Totally agree! Kills the flow. I mentioned the very issue myself today! Also, as a option, you could have a garage assigned to set it to "go-to closest". Something. I also found that looking for the garage/depot on a map is difficult too. Should have a different icon type and/or color.
